2020-02-01_cw/View/MainWindowView.xaml

```diff
@@ -0,0 +1,88 @@
+﻿<UserControl
+    x:Class="_2020_02_01_cw.View.MainWindowView"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="clr-namespace:_2020_02_01_cw"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:model="clr-namespace:_2020_02_01_cw.Model"
+    xmlns:view="clr-namespace:_2020_02_01_cw.View"
+    xmlns:viewmodel="clr-namespace:_2020_02_01_cw.ViewModel"
+    d:DesignHeight="450"
+    d:DesignWidth="800"
+    mc:Ignorable="d">
+
+    <UserControl.DataContext>
+        <viewmodel:MainWindowViewModel />
+    </UserControl.DataContext>
+
+    <UserControl.Resources>
+        <DataTemplate DataType="{x:Type model:Product}">
```

Comment:
It would be great if you moved controlds from DataTemplate to a separate control.